readme.md

```diff
@@ -10,7 +10,7 @@ An [ESLint shareable config](https://eslint.org/docs/developer-guide/shareable-c
 ## Usage
 
 ```
-npm install --save-dev eslint eslint-plugin-standard eslint-plugin-promise eslint-plugin-import eslint-plugin-node @typescript-eslint/eslint-plugin eslint-config-standard-with-typescript
+npm install --save-dev eslint@6 eslint-plugin-standard@4 eslint-plugin-promise@4 eslint-plugin-import@2 eslint-plugin-node@9 @typescript-eslint/eslint-plugin@1 eslint-config-standard-with-typescript
```

Comment:
For reference:

https://github.com/standard/eslint-config-standard-with-typescript/blob/ac8fa1bcff7a10d8d1cf2ea7e1767cc3ca512bda/package.json#L59-L67

The idea is that this command would install the latest compatible packages.

Tested in zsh, cmd and PowerShell 6.